keps/sig-storage/625-csi-migration/README.md

```diff
@@ -156,10 +163,211 @@ Running this set of tests is optional for a per-provider basis. We would
+  - For volume provision/delete:
+    PVC LIST/WATCH apis will be called by csi-provisioner to monitor the PVC status. PV CREATE api will be called by csi-provisioner to create PV. PV DELETE api will be called by csi-provisioner to delete PV. PVC/PV PATCH api will be called by csi-provisioner for updating the object.
+    Notice that these new calls from csi-provisioner also mean that we will reduce call from the kube-controller-manager side.
+  - When CSI driver is being installed, there will be CSIDriver/CSINode CREATE/PATCH api call by kubelet. For each kubelet that installs the driver there will be one call.
+  - csi-provisioner && csi-attacher will call one time GET api for CSINode object when provision/attach volume
+
+* **Will enabling / using this feature result in introducing new API types?**
+  No
+
+* **Will enabling / using this feature result in any new calls to the cloud provider?**
+  After switching to CSI driver model, all the volume operations including volume provision/deletion/attach/detach/mount/unmount/resize will be running through the CSI driver. 
+  So depending on how the CSI driver is designed and implemented, it could vary if there is any new calls being added. 
+  For example, `gce-pd` driver has the in-tree and CSI version of plugin implementation for all the operation mentioned above, once we switch from in-tree to CSI by CSI migration. 
+  If the implementation is the same, then there will not be new calls to the cloud provider. 
+  However, it is also possible that the plugin maintainer has different implementation so there might be new calls.
+
+* **Will enabling / using this feature result in increasing size or count of the existing API objects?**
+  - CSI migration will require CSI driver to be installed in the cluster so it can add CSI related API objects including CSIDriver, CSINode, VolumeAttachment. 
+  - The size of PV will include up to two more annotations including `volume.beta.kubernetes.io/storage-provisioner` and `volume.beta.kubernetes.io/migrated-to`. 
+  - The existing Node object will include new labels, specifically the CSI topology that are introduced by the CSI driver, e.g. `topology.gke.io/zone=us-central1-b` for GCE PD CSI Driver.
+  - For existing in-line volumes, there will be a new field under `VolumeAttachment.Spec.Source.VolumeAttachmentSource.InlineVolumeSpec` that will be populated if in-line volumes of migrated in-tree plugin is used.
+
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+  Depending on the design and implementation of the CSI Driver, the operation time taken could vary. 
+  In general, it might increase the total time spend because for the CSI sidecar to detect the object in the APIServer and do corresponding change through socket might add additional traffic compared to the in-tree plugin model.
```

Comment:
clarify the socket is a unix domain socket

Reply:
Done